src/notation/internal/notationinteraction.cpp

```diff
@@ -2976,15 +2972,10 @@ void NotationInteraction::updateGripEdit(const std::vector<Element*>& elements)
         resetGripEdit();
         return;
     }
-
     m_gripEditData.grips = element->gripsCount();
     m_gripEditData.curGrip = Ms::Grip::NO_GRIP;
     m_gripEditData.element = element;
-    m_gripEditData.grip.resize(m_gripEditData.grips);
-
     m_gripEditData.element->startEdit(m_gripEditData);
-    m_gripEditData.element->updateGrips(m_gripEditData);
```

Comment:
this was producing meaningless grip rectangles that weren't adjusted for current world transform at all